README.asciidoc

```diff
@@ -190,8 +190,8 @@ requests for consideration or create an issue with a code change proposal.
 === Installing dependencies
 
 On openSUSE one can install the package
-link:https://build.opensuse.org/project/show/devel:openQA[`os-autoinst-devel`] which
-pulls all dependencies.
+link:https://build.opensuse.org/project/show/devel:openQA[`os-autoinst-devel from devel:openQA`] which
+pulls in all dependencies. See also link:https://open.qa/docs/#_development_version_repository[`the openQA docs`].
```

Comment:
The link text shouldn't be formatted as code, remove the \` quotes.

Reply:
Why not? :)

Reply:
Because it isn't code or a command.

Reply:
That was in fact by accident :-D

Reply:
Oh I misread this and thought it was also about the previous link...